cmake/AddVSCode.cmake

```diff
@@ -0,0 +1,40 @@
+function(generate_vscode)
+    set(one_value_args "TARGET;FILE")
+    cmake_parse_arguments(args "${option_args}" "${one_value_args}" "" ${ARGN})
+
+    if(args_UNPARSED_ARGUMENTS)
+        message(SEND_ERROR "generate_vscode called with invalid arguments: ${args_UNPARSED_ARGUMENTS}")
+    endif()
+
+    if(NOT args_TARGET)
+        message(SEND_ERROR "generate_vscode: TARGET argument is missing")
+    endif()
+
+    if(NOT args_FILE)
+        message(SEND_ERROR "generate_vscode: FILE argument is missing")
+    endif()
+
+    # backup existing launch.json
+    if(EXISTS ${args_FILE})
+        string(TIMESTAMP t)
+        file(COPY_FILE ${args_FILE} ${args_FILE}.${t})
+    endif()
+
+    # collect all NED folders for given target
+    get_ned_folders(${args_TARGET} opp_run_ned_folders)
+    set(opp_run_ned_folders "\"-n$<JOIN:${opp_run_ned_folders},:>\"")
+
+    # collect libraries for opp_run
+    set(opp_run_libraries "")
+    _get_opp_run_dependencies(${args_TARGET} opp_run_dependencies)
```

Comment:
These are the OmnetC++ dependencies am I right?
Do they change frequently?

Reply:
These are the libraries of Artery (e.g. `libartery_core.so`, `libtraci.so`, ...). These might change when building Artery with other flags (e.g. `libartery_envmod.so` when WITH_ENVMOD is ON) or when you extend Artery with custom functionalities. Basically, they could change with every generation of the build system.

Reply:
What happens if one includes all the libs all the time in the `launch.json`? Will this work?